### ISSUE TITLE
[FUNC-CONCAT]: Not support function CONCAT

### ISSUE THREAD

**Issue by lignay** [matrixorigin/matrixone#1131]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 57dde60d5551b2502cd04bfaf7232396c387c746
- Hardware parameters:
- OS type: MacOS
- Others:
```

### Actual Behavior

mysql> select * from t1 where a like concat("abc","%");
ERROR 1105 (HY000): unknown error:[42000]'' is not support now

### Expected Behavior

mysql> select * from t1 where a like concat("abc","%");
+------+
| a    |
+------+
| abc  |
| abcd |
+------+
2 rows in set (0.00 sec)

### Steps to Reproduce

```markdown
create table t1 (a varchar(10));
insert into t1 values ("a"),("abc"),("abcd"),("hello"),("test");
select * from t1 where a like concat("abc","%");
```

**Comment by m-schen** [comment]
built-in function will not support at v0.2

**Comment by m-schen** [comment]
the same to issue #2111 , and will solved at #2111
